charts/fees-register-api/values.yaml

```diff
@@ -18,6 +19,8 @@ java:
     ISSUER_URI: https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net/o
     OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-{{ .Values.global.environment }}.internal:8443/openam/oauth2/hmcts
     OIDC_AUDIENCE_LIST: fees_admin_frontend,paybubble
+    SERVER_SHUTDOWN_STRATEGY: graceful
```

Comment:
you've also set this in your application configuration, consider whether duplicating this config is the right thing to do

Reply:
Even if right now values for application and values.yaml are same but going forward values.yaml will help us to changes the strategy.